ansible/roles/docker/tasks/debian-install.yml

```diff
@@ -1,7 +1,8 @@
 ---
 - name: DEBIAN | Make sure this is stretch or sid, jessies does not have docker
   fail: msg="Docker.io only available in sid and stretch, https://wiki.debian.org/Docker"
-  when: ansible_lsb.codename != "stretch" and ansible_lsb.codename != "sid"
+  when: ansible_distribution == "Debian" and
```

Comment:
Apologies. Leftover from when i was using the upstream docker.io package from ubuntu. Before i realized that it is incompatible with kubelet.